tests/ParserTest.php

```diff
@@ -1171,22 +1171,6 @@ public function flatCommentExtractingOneComment(): void
         self::assertCount(1, $comments);
         self::assertSame('Find Me!', $comments[0]->getComment());
     }
-    /**
-     * @test
-     */
-    public function flatCommentExtractingTwoComments(): void
-    {
-        $parser = new Parser('div {/*Find Me!*/left:10px; /*Find Me Too!*/text-align:left;}');
-        $document = $parser->parse();
-        $contents = $document->getContents();
-        $divRules = $contents[0]->getRules();
-        $rule1Comments = $divRules[0]->getComments();
-        $rule2Comments = $divRules[1]->getComments();
-        self::assertCount(1, $rule1Comments);
-        self::assertCount(1, $rule2Comments);
-        self::assertEquals('Find Me!', $rule1Comments[0]->getComment());
-        self::assertEquals('Find Me Too!', $rule2Comments[0]->getComment());
-    }
```

Comment:
It is worth keeping this test, but have it skipped until the correct fix is implemented?

Reply:
Done.